src/sas/sascalc/dataloader/readers/abs_reader.py

```diff
@@ -196,9 +203,12 @@ def get_file_contents(self):
                     # skip it.
                     pass
 
+            # SANS Data:
             # The 6 columns are | Q (1/A) | I(Q) (1/cm) | std. dev.
             # I(Q) (1/cm) | sigmaQ | meanQ | ShadowFactor|
-            if line.count("The 6 columns") > 0:
+            # USANS Data:
+            # EMP LEVEL: <value> ; BKG LEVEL: <value>
+            if line.count("The 6 columns") > 0 or line.count("EMP LEVEL"):
```

Comment:
Trivial point: ```line.index(target) >= 0``` will stop after the first target whereas ```line.count(target) > 0``` will continue to scan the entire line even after finding the target.  The efficiency difference is insignificant in this case since the line is short.  Even better is ```line.startswith(target)``` if target is guaranteed to be at the start of the line.